monkey/monkey_island/cc/ui/src/components/ui-components/AdvancedMultiSelect.js

```diff
@@ -1,121 +1,118 @@
-import React, {useState} from 'react';
+import React from 'react';
+import {Form} from 'react-bootstrap';
 
-import {Card, Button, Form} from 'react-bootstrap';
-import {FontAwesomeIcon} from '@fortawesome/react-fontawesome';
-import {faCheckSquare} from '@fortawesome/free-solid-svg-icons';
-import {faSquare} from '@fortawesome/free-regular-svg-icons';
 import {cloneDeep} from 'lodash';
 
 import {getComponentHeight} from './utils/HeightCalculator';
-import {resolveObjectPath} from './utils/ObjectPathResolver';
 import InfoPane from './InfoPane';
+import {MasterCheckbox, MasterCheckboxState} from './MasterCheckbox';
+import ChildCheckbox from './ChildCheckbox';
+import {getFullDefinitionByKey, getDefaultPaneParams} from './JsonSchemaHelpers.js';
 
+class AdvancedMultiSelect extends React.Component {
+  constructor(props) {
+    super(props)
```

Comment:
I think this is done by default, so not needed

Reply:
Removing `super(props)` causes runtime errors.